lona/html/abstract_node.py

```diff
@@ -58,6 +58,11 @@ def __eq__(self, other):
         if self.NODE_TYPE == NODE_TYPE.TEXT_NODE:
             return self._string == other._string
 
+        # widgets
+        # TODO: remove in 2.0
```

Comment:
Maybe issue a deprecation warning?

Reply:
Until now we did not issue deprecation warnings but added wrote them to the documentation. It's a good idea to deprecate the Legacy Widget API now, but I think it's a little bit much for this PR because the old API gets used in multiple examples in the documentation. I opened #437 as a base for a follow-up PR.